### PR TITLE
Centcom change, Deathsquad buff and Abductors get the new advanced camera console.

### LINKED ISSUE #8995: Emergency response team has no comms
I've played twice as an ERT member, the comms did not work both times. 
I'm unsure if the headsets use the station comms but that would be really inefficient as you don't usually have comms up when you call for an emergency response team. (It's also really hard to talk to your team)

I was under the impression they would have a channel like the deathsquad.

### PR DESCRIPTION
**Pull request by Fayrik**
So, first up. Centcom gets yet another redesign:
He's doing it again
Having cargo at the top was pretty dumb, but now I've utilized the extra room to add an "Administrative Office", which can be used for fun stuff with events and briefing ERTs/whatever.

And here's everything else:
- Abductors now get advanced camera consoles.
- Headsets can now be locked to a spesific frequency, this only occurs for nuke ops and deathsquads.
- The Centcom channel is now available over all z-levels, regardless of telecomms status.
- The Deathsquad space suits are now hardsuits.
- The Deathsquad now have thermal imaging huds, which can swap which hud they display on the fly.
- I've added myself to the admins.txt again because god damn it I hate having to add myself every time.

Fixes #8995

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:-1: until every floor tile in CC has a random gun on it

**Comment by Miauw62** [comment]
Could you try to expand the abductor ships and add some medical supplies, a recharger and a holding cell to them? The feedback thread has a neat design.

**Comment by Iamgoofball** [comment]
if the abductor ships don't look like a UFO you failed

**Comment by Fayrik** [comment]
Sure, @Miauw62. I noticed they're already a little bit bigger than they were before, but I'll see if I can give them more of a tear-drop shape.

**Comment by Fayrik** [comment]
Ayyliums

**Comment by Kromgar** [comment]
You should add some sort of Powersource sitting inside. Could be a unsimulated tile with a Supermatter Shard. Also add some damn wings or  something they look like boxes

**Comment by Miauw62** [comment]
theyre supposed to look like boxes they're ufos

**Comment by Fayrik** [comment]
You can't tell what they look like from the inside anyway, so adding any additional shape to them would only effect ghosts, and would probably look terrible anyway.

**Comment by Kromgar** [comment]
Still add a powersource that glows or something. To the inside so it looks fanceh. Also it needsa  tile that actually looks alien considering those are just AI tiles. It would be interesting to have a fleshy appearance or perhaps chrome

**Comment by Miauw62** [comment]
just copy some of the XCOM ships

**Comment by Iamgoofball** [comment]
doesn't feel UFO enough

**Comment by Supermichael777** [comment]
add indestructible windows. and some sort of fake controls.

**Comment by Alek2ander** [comment]
The scientist should be able to use the camera monitor and the teleportation console simultaneously. Running back and forth will be annoying.

**Comment by Xhuis** [comment]
A recharger wouldn't be a bad idea for the agent's pistol.

**Comment by GunHog** [comment]
Please put the camera console back next to the teleporter. It would be MAD annoying to have to move between them constantly.

**Comment by Fayrik** [comment]
@GunHog Where would you put it? My biggest gripe with the shuttle layout as it stands is the fact you have to move from the surgery table to you tools. Moving for a camera while scouting is one thing, moving while trying to keep someone restrained and operate on them is something very different.

In fact, I may have a solution, but I'm not sure if it'll work. Watch this space, I guess.

**Comment by GunHog** [comment]
There is no real point to using the table to store the tools anyway. The Scientist has only his Science tool, leaving plenty of space in his backpack to hold all the surgery implements. Unlike a human, the Scientist does not have to share them with the agent or anyone else, so keeping them in the backpack is no hindrance at all.

**Comment by Miauw62** [comment]
Yeah maybe it's a good idea to replace the topmost table with the advanced camera console. There's plenty of space for all the tools on the other two tables.

**Comment by Fayrik** [comment]
I get the joj done.
Honestly, at this rate, this will never get pulled.

Anyway, since I last updated the map, I actually played scientist, and the whole round was a revelation as I learned the strategy.
Between that and Miauw's suggestion (which I had thought of, but he said it so he gets the credit!), I now think this new layout could well be nearly perfect.
So why'd I make it smaller, anyway? Simple. Scientist stays on the left, agent on the right. There'll only be a few times where you have to swap sides, but the major point is that now you're basically stood on the same square the whole round, which is what abduction is really all about.
The only gripe I can see here is that there's no passing room, but if you kick out the racks and open the closets then you get a whole extra 5 tiles over your initial 8.

**Comment by Xhuis** [comment]
Oh my God, an __actual use__ for the previously-unused bed subtype? This gets all my +1's.

**Comment by Fayrik** [comment]
That's not just there for flavour, @Xhuis. I figured if a team ever captures more than one person at once, they'll need some way of restraining one person while working on the other.
I was hoping for a chair, but that looks way cooler.

**Comment by Xhuis** [comment]
Exactly! It's both useful and gives it a use.

**Comment by Fayrik** [comment]
Argh, aparently there's something special about the abductor camera consoles?
I'm not exactly sure what it is though, so I'm going to have to look into that.

**Comment by GunHog** [comment]
Abductor camera consoles work like the AI's eye. Instead of flipping though a list (which is very, very tedious and restricting), you will move a special camera object around.

**Comment by Fayrik** [comment]
@GunHog that wasn't the issue. This PR actually converts them from regular consoles to the new AI-style ones.
The issue I had was that I heard that you could use the current consoles to follow your agent while he's in the field. I had to double check that I hadn't accidentally deleted that feature.
(Spoiler: It's still there, we're all good.)

**Comment by GunHog** [comment]
Have you fixed https://github.com/tgstation/-tg-station/issues/9046 in this PR? The agent vest cameras having the same name means that you can only view one at a time. I recommend adding their designation to the camera tag.

Also, since the abductors cannot communicate with other ones from different ships, it may be wise to disallow marking another ship's agent. That, or allow them to communicate so they do not step on each other's toes, or in a more amusing situation, attack and abduct another ship's agent!

**Comment by Alek2ander** [comment]
HUD icons for abductor ayyliens of your team, other teams, and humans that are already processed.

**Comment by Fayrik** [comment]
@GunHog I haven't fix that, since this PR wasn't really meant to deal with any abductor code at all, I just tagged in a map edit on their ships to save the trouble of having to have two map edit PRs that could potentially conflict.
I could look at it in a seperate PR if you'd like, but really abductor is @AnturK 's work and I'd rather not interfere with it more than I already have.

**Comment by GunHog** [comment]
Wait, I did not notice this before: You changed the DS frequency from 1441 to 1337. I would like you to please keep that channel colored and with a name. Normal radios cannot reach down to that level.

**Comment by Miauw62** [comment]
oh no 1337 conflicts with random_frequency()! (which is just return 1337)

**Comment by GunHog** [comment]
"random".

**Comment by GunHog** [comment]
Whatever you did to comms, it is causing breaks in my script. I am not sure what yet, though.

**Comment by Fayrik** [comment]
When was the last test you did?
Might be #9095 related.

**Comment by GunHog** [comment]
I just tried it live on the server, was not broadcasting properly.  My script is complex, I will have to do extensive private code to find out.

File: code/game/machinery/telecomms/broadcaster.dm [hunk 1]
```diff
@@ -244,7 +244,20 @@ var/message_delay = 0 // To make sure restarting the recentmessages list is kept
...
+				continue
+
+			if(R.receive_range(freq, level) > -1)
+				radios += R
+
+	// --- This space left blank for Syndicate data ---
+
+	// --- Centcom radio, yo. ---
+
+	else if(data == 4)
```
Comment (GunHog) [inline code/game/machinery/telecomms/broadcaster.dm]: This, right here, along with the other one, is what broke NTSL. Data set to four is for *Artificial Transmissions*, I.E. signals sent by scripts. In addition, this bypasses normal tcomms signal logging due to this. I will need to change it in order to fix NTSL.

**Pull request merged**